livechat/agent/rtm/api/v33.py

```diff
@@ -11,8 +11,13 @@
 
 class AgentRtmV33:
     ''' Agent RTM API Class containing methods in version 3.3. '''
-    def __init__(self, url: str):
-        self.ws = WebsocketClient(url=f'wss://{url}/v3.3/agent/rtm/ws')
+    def __init__(
+        self,
+        url: str,
+        header: Union[list, dict, Callable, None],
```

Comment:
opt: use `typing.Optional`

> Optional[X] is equivalent to Union[X, None].

Reply:
It is copied from `WebSocketApp` client that is used in our SDK

```
class WebSocketApp:
    """
    Higher level of APIs are provided. The interface is like JavaScript WebSocket object.
    """

    def __init__(
        self,
        url: str,
        header: Union[list, dict, Callable, None] = None,
```